docs/tools/fbsql/fbsql-loaders-postgres.md

```diff
@@ -0,0 +1,99 @@
+---
+title: fbsql-loader-postgres
+layout: default
+parent: fbsql
+grand_parent: Tools
+nav_order: 2
+---
+
+# PostgreSQL Loader
+
+If fbsql is provided the `--loader-postgres=filename` flag, it will run in non-interactive mode. Based on the configuration provided in filename, fbsql will query PostgreSQL, read tuples returned, and submit them to FeatureBase via `BULK INSERT` statements. In this mode, fbsql processes messages until all tuples returned by the query are processed.
```

Comment:
Same as the kafka loader -- need to specify where the flag is found and list it under "Before you begin"